pkgs/development/tools/build-managers/conan/default.nix

```diff
@@ -20,6 +20,13 @@ let newPython = python3.override {
         inherit version;
         sha256 = "1vn1db2akw98ybnpns92qi11v94hydwp130s8753k6ikby95883j";
       };
+      patches = oldAttrs.patches or [] ++ [
+        # don't raise import error on non-linux os.
+        (fetchpatch {
+          url = "https://github.com/nir0s/distro/commit/25aa3f8c5934346dc838387fc081ce81baddeb95.patch";
+          sha256 = "0m09ldf75gacazh2kr04cifgsqfxg670vk4ypl62zv7fp3nyd5dc";
+        })
+      ];
```

Comment:
Maybe add a comment about removing the patch one distro can be upgraded to >=1.2.0.

Should be possible from conan 1.29.0 onwards.

Reply:
Fixed, thanks!